web/packages/teleport/src/AccessRequests/utils.ts

```diff
@@ -108,10 +108,10 @@ export function middleValues(
   }));
 }
 
-// Generate a list of middle values between now and the session TTL.
+// Generate a list of middle values between now and the request TTL.
 export function requestTtlMiddleValues(
   created: Date,
-  sessionTTL: Date
+  requestTTL: Date
```

Comment:
nit: while you're here, the convention in our TypeScript code is that we only capitalize the first letter of an initialism (which is why the function is named `requestTtlMiddleValue`.)
```suggestion
  requestTtl: Date
```